docs/permit-generation.md

```
@@ -0,0 +1,182 @@
+
+Custom split:
+
+```bash
+uv run python scripts/generate_permits.py --reset --correct 70 --minor 20 --major 10
+```
+
+## Filename scheme
+
+Every output filename encodes the flavor and (for errors) the mutation count:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)